### DISCUSSION TITLE
Possible to make it work as a 'Provider' in Bazarr?

### DISCUSSION THREAD

**Discussion by GOvEy1nw** [Purfview/whisper-standalone-win#221] [Q&A] [answered]
Hi, this is the easiest version I've whisper I've come across, and the included optional --standard argument really makes the subs more standardized compared to base Whisper outputs. For a more streamlined usage, I'd love to be able to use this as a 'Provider' inside of Bazarr, if that's something you could possibly look into? Bazarr has a 'Whisper' provider already, however it's a complicated setup, and doesn't include the standardized argument etc.

Many thanks!

**Comment by Purfview** [comment, marked as answer]
Bazarr is using some asr webservice, not the executables.

Anyway, these questions about Bazarr should be asked at the bazarr places:
https://github.com/morpheus65535/bazarr
https://discord.gg/MH2e2eb

Btw, you could launch whisper-faster.exe periodically on Radar/sonar download folder,  and use`--skip` arg.
